### DISCUSSION TITLE
Sound recording (feature request)

### DISCUSSION THREAD

**Discussion by aha999** [vmiklos/plees-tracker#233] [Ideas]
one crucial feature in sleep tracking app for me is recording the peaks in volume that you set & then in the morning you can listen to those peaks (if and when you were snoring, if a dog barking or traffic is waking you up, or any other noises etc.)
Please implement voice recorder, where you set the peaks where it starts recording, and then it can be reviewed on a sound line showing volume peaks and time.

**Comment by vmiklos** [comment]
It's a core behavior of this project that it does nothing while it's tracking your sleep, by default. This ensures it doesn't drain your battery.

However, if someone wants to add this feature in a way that you have to opt in, I don't mind. But it's a lot of work for little value, so I don't plan to add it myself.

Thanks for the report, anyhow. :-)
